save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/DemoStatus.kt

```diff
@@ -4,37 +4,38 @@ import kotlinx.serialization.Serializable
 
 /**
  * Enum that defines demo status
+ * @property statusTooltip
  */
 @Serializable
-enum class DemoStatus {
+enum class DemoStatus(val statusTooltip: String) {
     /**
      * Demo is created but stopped by error
      */
-    ERROR,
+    ERROR("Oops, some issue has occurred. Please, check logs and contact us."),
```

Comment:
> has occurred --> has been occurred?

if I am not mistaken, no

> I think instead of contact us it more appropriate to add link https://github.com/saveourtool/save-cloud/issues/new

Will do it